typescript/infra/test/warp-configs.test.ts

```diff
@@ -18,10 +18,9 @@ chai.should();
 const DEFAULT_TIMEOUT = 60000;
 
 const warpIdsToSkip = [
-  'EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-swell-taiko-zircuit',
```

Comment:
This is the old EZETH config so removing this since it's been merged

Reply:
Can we do that on a separate Pr?